.gitignore

```diff
@@ -88,6 +88,7 @@ logs/
 *.sln.docstates
 *.sln
 *.vcxproj*
+/custom.py
```

Comment:
It might be useful to add a comment above to explain why it's ignored:

```suggestion
# Custom SCons configuration override
/custom.py
```

Reply:
Technically it does override the default options, but options set from file can be overridden by command line options, so yes.